src/migrations/20240716135038-integration-events.js

```diff
@@ -0,0 +1,29 @@
+exports.up = function (db, cb) {
+    db.runSql(
+        `
+        CREATE TABLE IF NOT EXISTS integration_events
+        (
+            id SERIAL PRIMARY KEY NOT NULL,
```

Comment:
BIGSERIAL? Are we expecting more than 2B events here?

Reply:
Thanks, addressed in https://github.com/Unleash/unleash/pull/7604/commits/ed64b1f09afd6402b56a4da253829bfba38c0f4c